setup.py

```diff
@@ -8,7 +9,7 @@
         "requests"
     ],
     name="meilisearch",
-    version="0.18.0",
+    version=VERSION,
```

Comment:
I don't know if this works, is there a way to check this before accepting @alallema?

Reply:
Seems to work, or at least it doesn't cause an error.

```sh
python setup.py check
```

If you want to further verify you could publish this branch to Test PyPI https://test.pypi.org/.

I would suggest changing it to:

```py
version=__version__
```

Reply:
I don't know either, to be honest, I've never verified that it works that way. But like @sanders41 said, if it doesn't cause an error, it probably will.